tests/karma.conf.js

```diff
@@ -69,7 +67,6 @@ if (process.env.TEST_ENV === 'ci') {
     ]
   };
   karmaConf.reporters.push('coverage');
-  karmaConf.preprocessors['src/**/*.js'] = ['coverage'];
```

Comment:
Removing this line is on purpose, this is not an error, https://github.com/istanbuljs/babel-plugin-istanbul#karma is saying to not add the coverage preprocessor.
I compared the report coverage before and after this PR, this is working properly.